public/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
This file contains references to the code capsule app, you should consider replacing them with TODO values to avoid confusion